pkg/upgrade/upgrades/v25_2_set_ui_default_timezone.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2025 The Cockroach Authors.
+//
+// Use of this software is governed by the CockroachDB Software License
+// included in the /LICENSE file.
+
+package upgrades
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/cockroachdb/cockroach/pkg/clusterversion"
+	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
+	"github.com/cockroachdb/cockroach/pkg/ui"
```

Comment:
we could move `ui.DisplayTimezone` to a minimal standalone package e.g. `pkg/ui/uisetting` to avoid having pkg/upgrade/upgrades depend on pkg/ui just to get the setting def, which IIRC can be a hefty package to build?